diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index 1d708d859b7b8431a6849c5fe35c6f289d5e241f..0000000000000000000000000000000000000000
diff --git a/src/structures/ClientPresence.js b/src/structures/ClientPresence.js
index 5b448d78643b20fc1a25a0f51d29ccf062f25e05..07ad8d735ac87f4907312546a0bf850c2ad38327 100644
--- a/src/structures/ClientPresence.js
+++ b/src/structures/ClientPresence.js
@@ -56,6 +56,7 @@ class ClientPresence extends Presence {
         data.activities.push({
           type: activity.type,
           name: activity.name,
+          state: activity.state ?? undefined,
           url: activity.url,
         });
       }
@@ -64,6 +65,7 @@ class ClientPresence extends Presence {
         ...this.activities.map(a => ({
           name: a.name,
           type: a.type,
+          state: a.state ?? undefined,
           url: a.url ?? undefined,
         })),
       );
diff --git a/typings/index.d.ts b/typings/index.d.ts
index 766ff2deb2be077eb96b7a4cdc35e82650871a9e..3e0d513f83182c8876a245b8fac3881423a6bc83 100644
--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -4300,13 +4300,22 @@ export interface WebhookFields extends PartialWebhookFields {
 
 export type ActivitiesOptions = Omit<ActivityOptions, 'shardId'>;
 
-export interface ActivityOptions {
+interface StandardActivityOptions {
   name?: string;
   url?: string;
+  state?: string;
   type?: Exclude<ActivityType, ActivityType.Custom>;
   shardId?: number | readonly number[];
 }
 
+interface CustomActivityOptions {
+  name: string;
+  state: string;
+  type: ActivityType.Custom;
+}
+
+export type ActivityOptions = StandardActivityOptions | CustomActivityOptions;
+
 export interface AddGuildMemberOptions {
   accessToken: string;
   nick?: string;